akka-stream/src/main/resources/reference.conf

```diff
@@ -47,6 +47,9 @@ akka {
     # such as FileSource, FileSink and others.
     file-io-dispatcher = "akka.stream.default-file-io-dispatcher"
 
+    # Blocking IO like OutputStreamSource and InputStreamSink uses by default the same settings
+    blocking-io-dispatcher = "akka.stream.default-file-io-dispatcher"
```

Comment:
My own slightly presumptuous summary is that we should rename the `file-io-dispatcher` to `blocking-io-dispatcher` and use it for this as well (i.e. “what @ktoso said”).